fractions/javaee/remoting/src/main/java/org/wildfly/swarm/remoting/RemotingFraction.java

```diff
@@ -53,4 +54,14 @@ public RemotingFraction applyDefaults() {
 
         return this;
     }
+
+    public RemotingFraction port(int port) {
+        this.port.set( port );
+        return this;
+    }
+
+    public int port() {
+        return this.port.get();
+    }
+    private Defaultable<Integer> port = integer(4777);
```

Comment:
Default value of the legacy remoting port is 4447... and I see that this is wrong in Swarm already. JIRA it is.